.github/workflows/ci.yaml

```diff
@@ -0,0 +1,60 @@
+name: CI
+
+on: [push, pull_request]
+
+jobs:
+  run_tests:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+
+    - name: Use Node.js 
+      uses: actions/setup-node@v1  
+
+    - name: Installing Dredd
+      run: sudo npm install dredd --global --unsafe-perm=true --allow-root
+
+    - name: Installing node-fetch module
+      run: npm install node-fetch@2
+
+    - name: Installing hooks module  
+      run: npm install hooks
+
+    - name: Installing https module   
+      run: npm install https
+
+    - name: Installing file module 
+      run: npm install fs
+
+    - name: Build the stack
+      run: docker-compose up -d         
+      shell: bash
+      working-directory: ./test     
+
+    - name: Waiting for OpenSearch domain to be up.
+      run: |
+        counter=1
+        for counter in {1..10}
+        do
+          if [ $(curl -s -o /dev/null --head -w "%{http_code}" 'https://admin:admin@localhost:9200' -H 'Content-Type:application/json' --insecure -v) -ne 200 ]; then
+            sleep 30s
+          else
+            echo  "Waiting for addtional 30 seconds.. for Opensearch domain to be up."
+            sleep 30s
+            break 
+          fi  
+        done
+        if [ $counter -eq 11 ]; then
+          echo "Unable to connect with OpenSearch URL https://localhost:9200/"
+          exit 1
+        fi   
+      shell: bash
+      working-directory: ./test       
+   
+    - name: Run script file
+      run: |
+          export NODE_TLS_REJECT_UNAUTHORIZED=0
```

Comment:
Add a comment on why we need this

Reply:
We need to set NODE_TLS_REJECT_UNAUTHORIZED=0 for disabling certificate verification, Otherwise there be an error while testing our API for "MISSING TLS_CERTIFICATES".